htmd/parameterization/phi.py

```diff
@@ -108,37 +108,26 @@ def setPhi(coords, atoms, left, right, phi):
 
 def getPhi(coords, atoms):
```

Comment:
Can you remove the `getPhi` function and replace the calls with `dihedralAngle(mol.coords[dihedral, :, 0])` so we can merge this PR?